.github/workflows/ci.yml

```diff
@@ -112,7 +121,9 @@ jobs:
 
       - name: Install development tools
         run: |
+          # These workarounds are needed until PHIVE can do installs in a non-interactive way
           echo y | phive --no-progress update phpstan;
+          echo y | phive --no-progress update phpcs;
```

Comment:
```suggestion
          echo y | phive --no-progress update ${{ matrix.command }};
```
?

Reply:
Ah, the script name doesn’t equal the command name.

Reply:
Maybe unify the two?

Reply:
You mean, like merge into one command? Or parametrize the command?

Reply:
I meant: name the script the same as the command it’s calling, i.e. phpstan and phpcs instead of sniff and stan.

Reply:
I have now moved the tool name into the matrix as well so we can have keep the nice command names in the Composer scripts.

Reply:
Can matrix variables only be primitives? If not, the following could also work:

```yaml
matrix:
		command:
			- name: sniff
			  tool: phpcs
			- name: stan
			  tool: phpstan
```

and then use `${{ matrix.command.name }}` and `${{ matrix.command.tool }}`?

Reply:
Thanks! This would probably work, but I'm not really sure. I'll give it a try.

Reply:
No, this does not seem to work. I'll undo the corresponding change and repush.

Reply:
OK, thanks for trying, though.